feature-utils/poly-look/test/react-components/legends.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import {
+  BlockLegend,
+  LineLegend,
+} from "../../src/react-components/legends.jsx";
+
+/**
+ * @jest-environment jsdom
+ */
+const legendValues = [
+  ["No color", "#000000"],
+  ["Gray", "#888888"],
+  ["All colors", "#ffffff"],
```

Comment:
do we need these values if they're not used in tests atm?

Reply:
Yep, they're used. They're dummy legends.